wdl/CollectQcVcfWide.wdl

```diff
@@ -1,60 +1,78 @@
 version 1.0
 
-# Author: Ryan Collins <rlcollins@g.harvard.edu>
-
-# Workflow to gather SV VCF summary stats for an input VCF
+# Workflow to gather SV VCF summary stats for one or more input VCFs
 
 import "TasksMakeCohortVcf.wdl" as MiniTasks
 
-workflow ShardedQcCollection {
+workflow CollectQcVcfWide {
   input {
-    File vcf
+    Array[File] vcfs
+    Array[File] vcf_idxs
     String contig
     Int sv_per_shard
+    String? bcftools_preprocessing_options
     String prefix
 
-    File? vcf_idx
-
     String sv_base_mini_docker
     String sv_pipeline_docker
 
     # overrides for local tasks
+    RuntimeAttr? runtime_override_preprocess_vcf
     RuntimeAttr? runtime_override_collect_sharded_vcf_stats
     RuntimeAttr? runtime_override_svtk_vcf_2_bed
 
     # overrides for MiniTasks
-    RuntimeAttr? runtime_override_split_vcf_to_qc
+    RuntimeAttr? runtime_override_scatter_vcf
     RuntimeAttr? runtime_override_merge_subvcf_stat_shards
     RuntimeAttr? runtime_override_merge_svtk_vcf_2_bed
   }
 
-  # Tabix to chromosome of interest, and shard input VCF for stats collection
-  call MiniTasks.SplitVcf as SplitVcfToQc {
-    input:
-      vcf=vcf,
-      vcf_idx=vcf_idx,
-      contig=contig,
-      min_vars_per_shard=sv_per_shard,
-      prefix="vcf.shard.",
-      sv_base_mini_docker=sv_base_mini_docker,
-      runtime_attr_override=runtime_override_split_vcf_to_qc
+  String output_prefix = "~{prefix}.collect_qc_vcf_wide"
+
+  # Tabix each VCF to chromosome of interest, and shard input VCF for stats collection
+  scatter ( vcf_info in zip(vcfs, vcf_idxs) ) {
```

Comment:
Rework to use index:
```suggestion
  scatter ( i in range(length(vcfs)) ) {
```
so that you can have prefixes with shard number in them like `~{output_prefix}.scatter_vcf.shard_~{i}`